onnxscript/rewriter/ort_fusions/gelu.py

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+from __future__ import annotations
+
+import math
+
+from onnxscript import ir
+from onnxscript.rewriter import pattern
+
+_sqrt_two_over_pi = math.sqrt(2.0 / math.pi)
+
+
+class GeluTanhFusion(pattern.RewriteRuleClassBase):
+    def pattern(self, op, x):
+        # GELU(x) = 0.5 * x * {1 + Tanh[\sqrt(2/pi) * (x + 0.044715 * x^3)]}
+        t1 = op.Pow(x, 3)
+        t2 = op.Mul(0.044715, t1)
+        t3 = op.Add(x, t2)
+
+        t4 = op.Mul(_sqrt_two_over_pi, t3)
+        t5 = op.Tanh(t4)
+        t6 = op.Add(t5, 1)
+        t7 = op.Mul(x, t6)
+        result = op.Mul(0.5, t7)
```

Comment:
I found the reason that FastGelu does not match dynamo-based graph because of this. The order that we Mul(0.5) is reversed in dynamo-based graph. 

https://github.com/pytorch/pytorch/blob/ce54c430c0e9d5e6e9ee0b1d85bddd04fbcbca4e/torch/onnx/symbolic_opset9.py#L5668

https://github.com/microsoft/onnxscript/blob/b26817cf5ff863a229a2b47330c04cf0224cf77a/onnxscript/function_libs/torch_lib/ops/nn.py#L491

cc @justinchuby